src/projects/EnsureThat/Enforcers/StringArg.cs

```diff
@@ -37,6 +37,13 @@ public string IsNotNullOrEmpty([ValidatedNotNull]string value, [InvokerParameter
             return value;
         }
 
+        public string IsEmptyOrWhitespace(string value, [InvokerParameterName] string paramName = null, OptsFn optsFn = null)
+        {
+            if (value?.Trim() == "")
```

Comment:
`Trim()` is probably not the best for performance here, due to allocating a new string and requiring processing the entire input before giving a result rather than being able to short-circuit true on the first non-white-space character.

You may want to use the latest from the BCL which takes an approach like
```
        [Pure]
        public static bool IsNullOrWhiteSpace(String value) {
            if (value == null) return true;

            for(int i = 0; i < value.Length; i++) {
                if(!Char.IsWhiteSpace(value[i])) return false;
            }

            return true;
        }
```

Reply:
Implemented.